packages/rx_bloc_list/lib/src/widgets/rx_paginated_builder.dart

```diff
@@ -108,8 +145,9 @@ class RxPaginatedBuilder<B extends RxBlocTypeBase, T> extends StatefulWidget {
   /// the pull-down to refresh feature.
   factory RxPaginatedBuilder.withRefreshIndicator({
     required Stream<PaginatedList<T>> Function(B) state,
-    required Widget Function(BuildContext, AsyncSnapshot<PaginatedList<T>>, B)
-        builder,
+    required Widget Function(BuildContext, PaginatedList<T>, B) buildSuccess,
```

Comment:
Not really a request, but more of a comment on this. Maybe we could introduce a typedef for `Widget Function(BuildContext, PaginatedList<T>, B)` since it is more readable and used in several places.

Reply:
Looking at the other packages it seems that typedef introduces additional reading complexity, as the dev needs to go another level deeper to see all the details of the function signature.
As for now, we can leave it as it is.